fsw/pc-rtems/src/cfe_psp_exception.c

```diff
@@ -135,7 +135,7 @@ void CFE_PSP_ExceptionHook (int task_id, int vector, int32 *pEsf )
     /*
```

Comment:
Should this be strcpy?

Reply:
This code is merely a placeholder.  The exception processing in PC-RTEMS is not really implemented, and clearly bits and pieces were at one point copied from the vxworks psp.

My suggestion is to remove the entire block from lines 133-150, and just set CFE_PSP_ExceptionReasonString to something like `UNIMPLEMENTED`.

Note that the test in the `if` statement on line 140 is totally bogus because TaskName is a local stack array, it can never be NULL.